cmd/boulder-ra/main.go

```diff
@@ -50,6 +50,17 @@ type config struct {
 		// instructs the RA to reuse the previously created authz in lieu of
 		// creating another.
 		ReuseValidAuthz bool
+
+		// AuthorizationLifetime defines how long authorizations will be
+		// considered valid for. Given a value of 10 months when used with a 90-day
+		// cert lifetime, this allows creation of certs that will cover a whole
+		// year, plus a grace period of a month.
+		AuthorizationLifetime string
```

Comment:
We have a handy ConfigDuration type for this sort of thing. Unfortunately since the biggest unit for Durations is Hours, that means we have to express things in awkward hour units, like "2160h".

Instead, let's call this field AuthorizationLifetimeDays, and make it an int. Same below.